Tests/FluentAssertions.Net40.Specs/XmlElementAssertionSpecs.cs

```diff
@@ -0,0 +1,509 @@
+﻿using FluentAssertions.Formatting;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Xml;
+
+namespace FluentAssertions.Specs
+{
+    [TestClass]
+    public class XmlElementAssertionSpecs
+    {
+        private static XmlElement ParseElement(string xml)
+        {
+            var doc = new XmlDocument();
+            doc.LoadXml(xml);
+            return doc.DocumentElement;
+        }
+
+        #region BeEquivalent
+
+        [TestMethod]
+        public void When_asserting_xml_element_is_equivalent_to_another_xml_element_with_same_contents_it_should_succeed()
+        {
+            // This test is basically just a check that the BeEquivalent method
+            // is available on XmlElementAssertions, which it should be if
+            // XmlElementAssertions inherits XmlNodeAssertions.
+
+            //-------------------------------------------------------------------------------------------------------------------
+            // Arrange
+            //-------------------------------------------------------------------------------------------------------------------
+            var element = ParseElement(@"<user>grega</user>");
```

Comment:
I prefer not to use any private methods. They tend to get all kinds of magic eventually.